closure/goog/style/transform_test.js

```diff
@@ -100,7 +98,7 @@ function testIsSupported() {
 
 
 function testIs3dSupported() {
-  if (goog.userAgent.GECKO && !goog.userAgent.product.isVersion(10) ||
+  if (!goog.userAgent.product.isVersion(10) ||
```

Comment:
This doesn't make sense anymore - the right hand side of the || will only run if isVersion(10) but you && it with !isVersion(10), so it's always false.  This is the same thing as simply if (!isVersion(10))